beebop/versions.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from beebop import __version__
```

Comment:
consider

```
from beebop import __version__ as beebop_version
```

which will make this easier to understand later once we add poppunk as then you can do

```
from PopPUNK import __version__ as popunk_version
```